src/main/resources/anniv5Resources/localization/eng/coresetpack/Cardstrings.json

```diff
@@ -9,7 +9,7 @@
   },
   "${ModID}:DestinyDraw": {
     "NAME": "Destiny Draw",
-    "DESCRIPTION": "Choose 1 of 3 Packs. NL Gain a random card from the chosen Pack. It costs 0 this turn. NL Exhaust."
+    "DESCRIPTION": "Choose 1 of 3 random cards from different Packs to add to your hand. NL It costs 0 this turn. NL Exhaust."
```

Comment:
+1 for this change. The card itself being random made me never want to pick Destiny Draw after my first run using it.